virt/virt-getting-started.adoc

```diff
@@ -0,0 +1,80 @@
+:_content-type: ASSEMBLY
+[id="virt-getting-started"]
+include::_attributes/common-attributes.adoc[]
+= Getting started with {VirtProductName}
+:context: virt-getting-started
+
+toc::[]
+
+You can install and configure a basic {VirtProductName} environment in order to explore its features and functionality.
+
+[NOTE]
+====
+Cluster configuration procedures require `cluster-admin` privileges.
+====
+
+[id="planning-and-installing-virt_{context}"]
+== Planning and installing {VirtProductName}
+
+Plan and install {VirtProductName} on an {product-title} cluster:
+
+* xref:../installing/installing_bare_metal/preparing-to-install-on-bare-metal.adoc#virt-planning-bare-metal-cluster-for-ocp-virt_preparing-to-install-on-bare-metal[Plan your bare metal cluster for {VirtProductName}].
+* xref:../virt/install/preparing-cluster-for-virt.adoc#preparing-cluster-for-virt[Prepare your cluster for {VirtProductName}].
+* xref:../virt/install/installing-virt-web.adoc#virt-installing-virt-operator_installing-virt-web[Install the {VirtProductName} Operator].
+* xref:../virt/install/virt-enabling-virtctl.adoc#virt-enabling-virtctl[Install the `virtctl` command line interface (CLI) tool].
+
+[discrete]
+[id="additional-resources-planning-and-installing"]
+[role="_additional-resources"]
+=== Additional resources
+* xref:../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[Using a CSI-enabled storage provider].
+* xref:../virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc#virt-configuring-local-storage-for-vms[Configuring local storage for virtual machines].
+* xref:../networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc#installing-the-kubernetes-nmstate-operator-cli[Installing the Kubernetes NMState Operator].
+* xref:../virt/virtual_machines/advanced_vm_management/virt-specifying-nodes-for-vms.adoc#virt-specifying-nodes-for-vms[Specifying nodes for virtual machines].
+* xref:../virt/virt-using-the-cli-tools.adoc#virt-virtctl-commands_virt-using-the-cli-tools[`Virtctl` commands].
+
+[id='configuring-networks_{context}']
+== Configuring secondary networks
```

Comment:
I created it as level 2 because I did not see configuration as a subsection of "Planning and Installing".

However, I agree that because we do not have any other configuration, it looks a bit odd. I will think about this.